goth_tests/pyproject.toml

```diff
@@ -22,7 +22,7 @@ pytest = "^6.2"
 pytest-asyncio = "^0.14"
 #goth = "^0.11"
 # to use development goth version uncomment below
-goth = { git = "https://github.com/golemfactory/goth.git", rev = "51e5b6f7f8e76b2f7b6f572e3ef127b0618281f4" }
+goth =  { git = "https://github.com/golemfactory/goth.git", rev = "6061a48342213c7a30e643ab0d86a58abdab460a" }
```

Comment:
Maybe bump it to latest commit - it was supposed to make test start faster https://github.com/golemfactory/goth/commit/5244075a4742cfdc848b8dfc4084f788be8b2adf